### PR TITLE
feat: price impact excludes fees

### PR DESCRIPTION
**Pull request by Voyz**
- added util functions for fee calculations
- price impact calculation now excludes fees

Based on discussion in: https://github.com/uniswap-python/uniswap-python/issues/310

### CODE REVIEW

File: uniswap/util.py [hunk 1]
```diff
@@ -126,3 +126,20 @@ def nearest_tick(tick: int, fee: int) -> int:
 def chunks(arr: Sequence[Any], n: int) -> Generator:
     for i in range(0, len(arr), n):
         yield arr[i : i + n]
+
+def fee_to_fraction(fee:int):
+    return fee / 1000000
+
+def fraction_to_fee(fraction:float):
```
Comment (ErikBjare) [inline uniswap/util.py:133]: Unused

**Review by ErikBjare — APPROVED** [review]
Looks good!

**Comment by Voyz** [comment]
Hey @ErikBjare removed the function and formatted with black 👍

**Comment by Voyz** [comment]
@ErikBjare just wanted to bring this PR to your attention again

**Comment by ErikBjare** [comment]
Very nice, thanks for the ping!

**Comment by ErikBjare** [comment]
And thanks for contributing! ❤️

**Comment by Voyz** [comment]
Glad to have helped! 🙌

**Pull request merged**